partitura/musicanalysis/note_array_to_score.py

```diff
@@ -0,0 +1,275 @@
+import partitura.score
+from partitura.score import ScoreLike, Part
+from partitura.utils import (estimate_symbolic_duration, estimate_clef_properties, key_name_to_fifths_mode, fifths_mode_to_key_name)
+import warnings
+import numpy as np
+from typing import Union
+import numpy.lib.recfunctions as rfn
+from fractions import Fraction
+import partitura.musicanalysis as analysis
+import partitura.score as score
+
+
+def create_divs_from_beats(note_array):
+    duration_fractions = [Fraction(float(ix)).limit_denominator(256) for ix in note_array["duration_beat"]]
+    onset_fractions = [Fraction(float(ix)).limit_denominator(256) for ix in note_array["onset_beat"]]
+    divs = np.lcm.reduce(
+        [Fraction(float(ix)).limit_denominator(256).denominator for ix in np.unique(note_array["duration_beat"])])
+    onset_divs = list(map(lambda r: int(divs * r.numerator / r.denominator), onset_fractions))
+    min_onset_div = min(onset_divs)
+    if min_onset_div < 0:
+        onset_divs = list(map(lambda x: x - min_onset_div, onset_divs))
+    duration_divs = list(map(lambda r: int(divs * r.numerator / r.denominator), duration_fractions))
+    na_divs = np.array(list(zip(onset_divs, duration_divs)), dtype=[("onset_div", int), ("duration_div", int)])
+    return rfn.merge_arrays((note_array, na_divs), flatten=True, usemask=False)
+
+
+def create_part(
+        ticks,
+        note_array,
+        key_sigs,
+        part_id=None,
+        part_name=None,
+        sanitize=True,
+        anacrusis_divs=0
+):
```

Comment:
Maybe we could add some documentation to this function? (or point to `note_array_to_score` for info on the arguments. (Also, it would be great to add typing annotations in this module in general).